StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/NamingRules/SA1308CodeFixProvider.cs

```diff
@@ -66,16 +60,24 @@ public override FixAllProvider GetFixAllProvider()
                     break;
                 }
 
-                if (!string.IsNullOrEmpty(token.ValueText))
+                if (numberOfCharsToRemove == 0)
+                {
+                    continue;
```

Comment:
Awesome. BTW, I asked because I was using this pull request to demonstrate to someone how code coverage tools can be used to ask more specific questions during code review.